solidity/contracts/system/KeepFactorySelection.sol

```diff
@@ -34,28 +38,38 @@ library KeepFactorySelection {
 
         KeepFactorySelector factorySelector;
 
-        // Standard ECDSA keep factory: KEEP stake and ETH bond.
+        // Standard ECDSA keep vendor and factory: KEEP stake and ETH bond.
         // Guaranteed to be set for initialized factory.
+        IBondedECDSAKeepVendor keepStakeVendor;
         IBondedECDSAKeepFactory keepStakeFactory;
 
-        // Fully backed ECDSA keep factory: ETH stake and ETH bond.
+        // Fully backed ECDSA keep vendor and factory: ETH stake and ETH bond.
+        IBondedECDSAKeepVendor ethStakeVendor;
```

Comment:
`fullyBackedFactoryVendor` perhaps? We do not have ETH stake.

Reply:
5f95e34cfbdf1aeb14b89995d03854fea7bfef62